webots/controllers/Supervisor/supervisor.py

```diff
@@ -150,7 +122,8 @@ def main_loop():
     supervisor_com_rx = supervisor.getDevice(SUPERVISOR_RX_NAME)
 
     if supervisor_com_rx is None:
-        print(f"No supervisor communication possible, because {SUPERVISOR_RX_NAME} not found.")
```

Comment:
line length limit of pylint should be configurable. 79 chars is not really useful, do we want to stay 100% PEP8 compliant?